holopy/inference/tests/test_interface.py

```diff
@@ -44,7 +44,7 @@ class TestUserFacingFunctions(unittest.TestCase):
     def test_cannot_sample_without_model(self):
         self.assertRaises(ValueError, sample, DATA, Sphere())
 
-    @attr('fast')
+    @attr('slow')
```

Comment:
@barkls, is there an easy way to make this slow test run faster?

Reply:
Yes. We aren't actually trying to test HoloPy's interactions with emcee here, just that the handy interface works correctly. This test is really doing 2 things and they should be broken up.

1) Check that the high-level `sample` function calls the `strategy.sample` method. This can be made fast by defining a dummy test strategy with a `sample` method that doesn't actually sample with emcee.

2) Check that the high-level `sample` function uses the right strategy if none is provided. This is actually redundant with another test further down in the file (`StrategyHandling.test_default_sampling_strategy_is_emcee` - should really be renamed `TestStrategyHandling`) so we can remove it here but it's probably just as slow there. One way to speed this operation up in the other test might be to design the sampling to fail with a pathological data or model. We could then check an `assertRaises` from within the `EmceeStrategy.sample` method to make sure it directs to the right place, but doesn't get as far as actually sampling.

Thoughts? Let me know if anything I wrote is unclear...